webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -150,166 +147,147 @@ class ManageManifestModal extends Component {
-          <button
-            className="close"
-            onClick={this.hideModal}
-            aria-label={__('Close')}
-          >
-            <Icon type="pf" name="close" />
-          </button>
-          <Modal.Title>{__('Manage Manifest')}</Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <Tabs id="manifest-history-tabs">
-            {showManifestTab &&
-              <Tab eventKey={1} title={__('Manifest')}>
-                <Form className="form-horizontal">
-                  {showRedHatProviderDetails &&
-                    <React.Fragment>
-                      <h5>{__('Red Hat Provider Details')}</h5>
-                      <hr />
-                      <FormGroup>
-                        <Col sm={3}>
-                          <ControlLabel htmlFor="cdnUrl">
-                            {__('Red Hat CDN URL')}
-                          </ControlLabel>
-                        </Col>
-                        <Col sm={9}>
-                          <FormControl
-                            id="cdnUrl"
-                            type="text"
-                            defaultValue={this.state.redhat_repository_url || organization.redhat_repository_url || ''}
-                            onBlur={this.updateRepositoryUrl}
-                          />
-                        </Col>
-                      </FormGroup>
-                      <FormGroup>
-                        <Col smOffset={3} sm={3}>
-                          <Button onClick={this.saveOrganization} disabled={organization.loading}>
-                            {organization.loading ? buttonLoading : __('Update')}
-                          </Button>
-                        </Col>
-                      </FormGroup>
-                      <br />
-                    </React.Fragment>
-                  }
-                  {showSubscriptionManifest &&
-                    <React.Fragment>
-                      <h5>{__('Subscription Manifest')}</h5>
-                      <hr />
-
-                      <FormGroup>
-                        <ControlLabel
-                          className="col-sm-3 control-label"
-                          htmlFor="usmaFile"
-                          style={{ paddingTop: '0' }}
-                        >
-                          <OverlayTrigger
-                            overlay={
-                              <Tooltip id="usma-tooltip">
-                                {__('Upstream Subscription Management Application')}
-                              </Tooltip>
-                            }
-                            placement="bottom"
-                            trigger={['hover', 'focus']}
-                            rootClose={false}
-                          >
-                            <span>{__('USMA')}</span>
-                          </OverlayTrigger>
+      <ForemanModal id={MANAGE_MANIFEST_MODAL_ID} title={__('Manage Manifest')}>
+        <ForemanModal.Header />
```

Comment:
Is this still needed after the change in foreman core?

Reply:
@MariaAga no, updating